cloud_functions/api_router/main.py

```
@@ -448,0 +1,448 @@
+            payload=payload,
+            auth_header_present="Authorization" in headers,
+            headers=headers,
+        )
+
+        # For token refresh, we need to pass the data_source as a query parameter
+        if "data_source" in payload:
+            url = f"{url}?data_source={payload['data_source']}"
+            partial_log_structured(message="Modified URL with query params", url=url)
+
+        # Make the request
+        response = requests.post(url=url, json=payload, headers=headers)
```

Ruff (0.8.2)

Probable use of `requests` call without timeout

(S113)